src/dndbeyond/content-scripts/character.js

```diff
@@ -907,7 +909,8 @@ async function rollItem(force_display = false, force_to_hit_only = false, force_
             brutal,
             force_to_hit_only,
             force_damages_only,
-            {weapon_damage_length});
+            {weapon_damage_length},
+            is_versatile);
```

Comment:
Actually, it might be better not to pass `is_versatile` to buildAttack, since it's a thing that is specific to weapon attacks, not a generic attack thing, so I'd rather see it added afterwards (like we do for `item-type` below), otherwise it wouldn't make sense for us to see a spell attack with `request.versatile = false`

Reply:
That's 110% the right approach I think, code pushed, testing now.